### PR TITLE
Updated the table

### PR DESCRIPTION
**Pull request by Satakshijain**
**Description**
Updated the page by by removing Nikhil Ladha’s entry, adding Archit Sharma under “Layer5 Documentation” and adding Yash Sharma under “Layer5 Cloud”

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by vishalvivekm — APPROVED** [review]
LGTM.
Thank you @Satakshijain

**Comment by ritiksaxena124** [comment]
Thank you @Satakshijain for quickly working on this PR 🚀. I have added the name of Yash Sharma as the maintainer of Layer5 cloud backend that was missed. 

Once the preview is done we can merge this PR

**Comment by l5io** [comment]
🚀 Preview for commit 5a5c60953a39dcdb1e963f401c9809275766afa1 at: https://65cda761fbdf026f96583a22--layer5.netlify.app

**Pull request merged**
